frontend/src/pages/beta/accounts/[...slug].tsx

```diff
@@ -82,18 +90,27 @@ const AccountQueryView: React.FC<QueryProps> = React.memo((props) => {
 
 export const getServerSideProps: GetServerSideProps<
   {},
-  { id: string }
+  { slug: string[] }
```

Comment:
Do we have to give up on Next.js routing to avoid the account header re-render when user navigates between the tabs? It seems that recently added [Nested Layouts feature in Next.js](https://nextjs.org/blog/layouts-rfc) should help here